.github/workflows/ci.yaml

```diff
@@ -95,9 +94,17 @@ jobs:
         if: steps.cache-cargo.outputs.cache-hit != 'true'
         run: RUSTC_BOOTSTRAP=1 cargo install grcov
 
+      - name: Install Rust toolchain nightly
```

Comment:
Why install Rust stable if we then install Rust nightly over it? Wouldn't it be better to just revert to installing Rust nightly at the beginning?

Reply:
We install stable, compile grcov with this toolchain, then install nightly to build clarinet with profiling enabled.
I tried what you suggested, using the profiler enabled in the Cargo.toml with stable but ended up with an empty lcov file, so ended up keeping the nightly approach.

Reply:
@lgalabru But it doesn't look like that's what's happening here:
https://github.com/hirosystems/clarinet/blob/48e8f62bdb67518328725b43912eccd334e22f92/.github/workflows/ci.yaml#L74-L103

Rust stable is being installed, cache restored, grcov installed, then Rust nightly is being installed over Rust stable, _then_ we compile and test. I think this would be equivalent to installing Rust nightly in the beginning instead of Rust stable, wouldn't it?

Reply:
when we invoke the command `cargo install grcov` we're not pulling a pre-built binary, grcov is being compiled with the stable toolchain that just installed.

Reply:
@lgalabru Correct, so why not install just `grcov` onto Rust nightly? What are we gaining by having this circuitous approach?

Reply:
Feel free to revisit this flow! Getting the CI back on track ASAP was important for increasing our confidence in the upcoming release, if we could just stop relying on nightly that'd be great.

Reply:
> we recently switched to rust `edition2021` and had to upgrade our version of rust + rust nightly and installing `grcov` using more recent nightlies was failing (https://github.com/hirosystems/clarinet/runs/5726898694?check_suite_focus=true).

Oh gotcha, yeah if it fails to install grcov using Rust nightly I can totally see why this is necessary.